services/supervisor/src/simple/mod.rs

```diff
@@ -14,33 +14,31 @@
 
 //! A simple supervisor service that does actions without any confirmations.
 
-pub use proto::ConfigPropose;
-pub use schema::Schema;
-
 use exonum::{
     blockchain::{self, InstanceCollection},
     crypto::Hash,
     helpers::ValidateInput,
     merkledb::Snapshot,
     runtime::{
-        rust::{interfaces::ConfigureCall, BeforeCommitContext, Service, TransactionContext},
-        Caller, ConfigChange, DispatcherError, ExecutionError, InstanceDescriptor, InstanceId,
+        rust::{CallContext, Service},
+        Caller, DispatcherError, ExecutionError, InstanceDescriptor, SUPERVISOR_INSTANCE_ID,
     },
 };
 use exonum_derive::{exonum_service, IntoExecutionError, ServiceFactory};
 
-mod proto;
+use crate::{update_configs, ConfigChange, ConfigPropose, ConfigureCall};
+
 mod schema;
-#[cfg(test)]
-pub mod tests;
+pub use self::schema::Schema;
 
+/// Simple supervisor for testing purposes. **This supervisor is not fit for real-world use cases.**
 #[derive(Debug, ServiceFactory)]
 #[exonum(
-    proto_sources = "proto::schema",
-    artifact_name = "exonum-testkit-supervisor",
-    implements("Transactions")
+    proto_sources = "crate::proto",
+    artifact_name = "simple-supervisor",
+    implements("SupervisorInterface")
 )]
-pub struct SimpleSupervisor;
+pub struct Supervisor;
```

Comment:
By the way I don't really like this rename, since now we have to remember current directory to understand that it's not a main supervisor, but a simple one. IMO it's a bit confusing.